webtau-feature-testing/src/main/groovy/com/twosigma/webtau/featuretesting/WebTauTestRunner.groovy

```diff
@@ -71,6 +71,7 @@ class WebTauTestRunner implements StepReporter, StandaloneTestListener {
             args.add(testPath.toString())
 
             WebTauConfig.resetConfigHandlers()
+            getCfg().reset()
```

Comment:
I had to add this because I found that in the features test, cfg was getting preserved between tests.  I think we've just gotten lucky before but in my case the disabled redirects test ran first and then passed, redirects on ran next but cfg still had it disabled.